src/style.css

```diff
@@ -1 +1,121 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.page {
+  font-family: Avenir, sans-serif;
```

Comment:
Good job on including a fallback font with the `font-family` property. This ensures that if the primary font fails to load, the text will still be displayed in a different font.